src/package_io/parse_input.star

```diff
@@ -95,7 +89,9 @@ def parse_input(plan, input_args):
             )
         )
 
-    result["tx_spammer_params"] = get_default_tx_spammer_params()
+    if "tx_spammer_params" not in result:
+        result["tx_spammer_params"] = get_default_tx_spammer_params()
```

Comment:
It'd overwrite all params, even those unspecified. If user does not specify all params, the command will fail. The code will prevent that: 

```suggestion
    result["tx_spammer_params"] = dict(get_default_tx_spammer_params(), **result["tx_spammer_params"])
```

Reply:
Applied, thanks!